core/ops.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+use crate::CoreOp;
+use crate::CoreOpHandler;
+use crate::Op;
+use crate::OpId;
+use crate::PinnedBuf;
+use std::collections::HashMap;
+
+#[derive(Default)]
+pub struct OpRegistry {
+  pub ops: Vec<Box<CoreOpHandler>>,
+  pub phone_book: HashMap<String, OpId>,
```

Comment:
100% bikeshedding but maybe we should either leave comment on this or rename `phone_book` to something else 😅